vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
```

Comment:
Two things wrong here:

* You already the location of the vars (the root of transmart-data), you certainly don't need to have the user point it out..
* Adding a directory to PATH where you don't need its flexibility (because you know where the executables are) is just sloppy and unnecessarily confusing.